jsconfig.json

```diff
@@ -2,9 +2,9 @@
   "compilerOptions": {
     "baseUrl": ".",
     "paths": {
-      "@/*": ["/*"],
-      "@sme/*": ["/lib/sme/*"],
-      "@backend/*": ["/backend/*"]
+      "@/*": ["./*"],
+      "@sme/*": ["./lib/sme/*"],
+      "@backend/*": ["./backend/*"]
```

Comment:
FYI, the direction is for us to make things simple and just remove all aliases except for the `"@/*": ["./*"]` one.